code/modules/core_implant/cruciform/machinery/armaments.dm

```diff
@@ -236,9 +236,3 @@
 	cost = 50
 	min_cost = 25
 	path = /obj/item/computer_hardware/hard_drive/portable/design/nt/new_guns
-
-/datum/armament/item/gun/firstaid
```

Comment:
Why removing thiss?

Reply:
It's a noob trap, there isn't ever a reason to order a medkit that is almost same cost as something like a purger.

Reply:
Fair